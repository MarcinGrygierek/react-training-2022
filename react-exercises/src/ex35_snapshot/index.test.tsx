import renderer from 'react-test-renderer';
import { Alert } from '.';

describe('Alert', () => {
    test('error renders correctly', () => {
        const tree = renderer.create(<Alert type='error' text='Lorem ipsum' />).toJSON();
        expect(tree).toMatchSnapshot();
    })
    test('success renders correctly', () => {
        const tree = renderer.create(<Alert type='success' text='Lorem ipsum' />).toJSON();
        expect(tree).toMatchSnapshot();
    })
})
