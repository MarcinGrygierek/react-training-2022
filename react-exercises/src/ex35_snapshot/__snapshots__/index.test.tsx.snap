// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert error renders correctly 1`] = `
<div
  className="alert alert-error"
  style={
    Object {
      "backgroundColor": "red",
      "padding": 20,
    }
  }
>
  Lorem ipsum
</div>
`;

exports[`Alert success renders correctly 1`] = `
<div
  className="alert alert-success"
  style={
    Object {
      "backgroundColor": "green",
      "padding": 20,
    }
  }
>
  Lorem ipsum
</div>
`;
